modules/dnn/test/test_model.cpp

```diff
@@ -221,6 +221,28 @@ TEST_P(Test_Model, DetectionMobilenetSSD)
                     scoreDiff, iouDiff, confThreshold, nmsThreshold, size, mean, scale);
 }
 
+TEST_P(Test_Model, Detection_normalized)
+{
+    std::string img_path = _tf("grace_hopper_227.png");
+    std::vector<int> refClassIds = {15};
+    std::vector<float> refConfidences = {0.999222f};
+    std::vector<Rect2d> refBoxes = {Rect2d(0, 4, 227, 222)};
+
+    std::string weights_file = _tf("MobileNetSSD_deploy.caffemodel");
+    std::string config_file = _tf("MobileNetSSD_deploy.prototxt");
+
+    Scalar mean = Scalar(127.5, 127.5, 127.5);
```

Comment:
Maybe `Scalar mean{127.5, 127.5, 127.5};` ?

Reply:
I think we can replace it later, thanks!